gatsby-node.js

```diff
@@ -11,7 +11,7 @@ const { paginate } = require("gatsby-awesome-pagination");
 const { createFilePath } = require("gatsby-source-filesystem");
 const config = require("./gatsby-config");
 
-if (process.env.CI) {
+if (process.env.CI === "true") {
```

Comment:
Isn't the `CI` property a bool?

Reply:
Hi @Nikhil-Ladha,

Yes, I thought so also, but when I tried to set the CI value to `false`, the CI conditional still executed as `true`, and then some googling showed and using `typeof`: 
- environment variables are always set as strings. ([src](https://torvo.com.au/articles/4-problems-with-environment-variables#:~:text=Environment%20variables%20are%20always%20set,parse%20it%20from%20a%20string.))
- when you set an env variable to `false`, e.g. DEBUG=false, it evaluates as the string "false" ([src](https://rusingh.com/all-been-there-boolean-environment-variables/))

So I had to update all the usages of the CI variable (since it was relevant to a Lighthouse (LHCI) issue).

Thanks,
Randy

Reply:
Interesting find!